homework/english/BuhtiyarovP_hw_eng_1/lesson1/grammar.md

```diff
@@ -0,0 +1,49 @@
+# Упражнения Present Simple и Present Continuous
+
+### Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+1. Your mum (work) today?                            --- Is your mum working today?
+2. Why this boys (to talk) now?                      --- Why are this boys talking now?
+3. Where he (to live)?                               --- Where does he live?
+4. Their mother (cook) very well?                    --- Does their mother cook very well?
+5. Lili is not rich, she (not to drive) a Mercedes.  --- Lili is not rich, she doesn't drive a Mercedes.
+6. Let´s go out. It (not rain) now.                  --- Let's go out. It isn't raining now.
+7. I have to go now. It (get) dark                   --- I have to go now. It is geting dark.
+8. A young man usually (to stand) at the window.     --- A young man usually stands at the window.
+9. Ann usually (make) doll dresses herself.          --- Ann usually makes doll dresses herself.
+10. His wife (have) coffee at the moment?            --- Is his wife having coffee at the moment?
+11. Let’s go for a walk. It (not/rain).              --- Let's go for a walk. It isn't raining.
+12. Look! The boys (come) home.                      --- Look. The boys are coming home.
+13. Maila (like) Chinese food?                       --- Does Maila like Chinese food?
+14. My grandmother (not to work). She is on pension. --- My grandmother doesn't work.
+15. What your brother (watch) now?                   --- What is your brother watching now?
+16. Where you (live)?                                --- Where do you live?
+17. You (to play) the piano well?                    --- Do you play the piano well?
+18. Where their brother (swim) now?                  --- Where is their brother swiming now?
+19. What they (to eat) at   breakfast?               --- What do they eat at breakfast?
+20. They (to swim) at the moment?                    --- Are they swiming at the moment?
+21. Mike (not wear) a tie today.                     --- Mike isn't wearing a tie today.
+22. Maila (not to watch) TV every morning.           --- Maila doesn't watch TV every morning.
+23. Listen! Blanca (sing) in the bathroom.           --- Listen. Blanca is singing in the bathroom.
+24. Bea sometimes (read) comics.                     --- Bea sometimes reads comics.
+
+### Найдите ошибки и исправьте их.
+1. We not going to school today.                     --- We aren't going to school today.
+2. What you doing after school?                      --- What are you doing after school?
+3. At the moment Peter is work in Russia.            --- At the moment Peter is working in Russia.
+4. Does he got a new car?                            --- Does he got a new car?
+5. He never wear a hat.                              --- He never don't wear a hat.
```

Comment:
This is no Russian!
Use or "never", either "don't"
You should write "He never wears a hat."